source/Nevermore/CommandExecutor.cs

```diff
@@ -258,7 +258,7 @@ void DisposeOfParameters()
                 // Parameters can contain streams and other disposable objects.
                 if (parameter.Value is IDisposable disposable)
                 {
-                    disposable.Dispose();
+                    //disposable.Dispose();
```

Comment:
We can dispose of these at the transaction level once we are done. Just left it like this as it is only a PoC